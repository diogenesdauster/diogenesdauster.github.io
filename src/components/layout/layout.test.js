import React from 'react'
import renderer from 'react-test-renderer'
import Layout from './layout'

describe('Layout', () =>
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Layout title={'About'}>
          <p>Hello Wolrd</p>
        </Layout>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  }))
