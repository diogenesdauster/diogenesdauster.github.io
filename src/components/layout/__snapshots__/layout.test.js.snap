// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
Array [
  <div>
    Loading (StaticQuery)
  </div>,
  <div
    className="content container"
  >
    <p>
      Hello Wolrd
    </p>
  </div>,
]
`;
